test/e2e/quadlet/host.container

```diff
@@ -0,0 +1,7 @@
+[Container]
+Image=localhost/imagename
+## assert-podman-args "--add-host=my-host-name:192.168.10.10"
+## assert-podman-args "--add-host=my-second-host-name:192.168.10.10"
+AddHost=my-host-name;my-second-host-name:192.168.10.10
+## assert-podman-args "--add-host=my-third-host-name:192.168.10.11"
+AddHost=my-third-host-name:192.168.10.11
```

Comment:
Add an empty line at the end of the file

Reply:
done